testing/resources/kind_config.yaml

```diff
@@ -0,0 +1,31 @@
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+nodes:
+  - role: control-plane
+  - role: worker
+    extraPortMappings:
+      - containerPort: 30080
+        hostPort: 8003
+      - containerPort: 31280
+        hostPort: 8004
+kubeadmConfigPatches:
+  - |
+    apiVersion: kubelet.config.k8s.io/v1beta1
```

Comment:
Not relevant to this PR specifically but just realised we should try to move away from v1beta1 as it was dropped from v1.16 onwards